packages/core/EventManager.ts

```diff
@@ -186,9 +186,12 @@ export default class EventManager {
         result.type = result.createdEvent.type;
         //responses data is later sent to webhook
         if (evt.location && evt.responses) {
-          evt.responses["location"].value = {
-            optionValue: "",
-            value: evt.location,
+          evt.responses["location"] = {
+            ...(evt.responses["location"] ?? {}),
+            value: {
+              optionValue: "",
+              value: evt.location,
+            },
```

Comment:
When an event type doesn't have any location set and user submits booking form there is no location field in responses causing Type error "Cannot set properties of undefined"